18-Oct-19 18:09:44 - Job Started : 2019-10-18 18:09:44
18-Oct-19 18:09:48 - User logged in and created SparkContext and SparkSession
18-Oct-19 18:09:48 - Feteched command line arguments bank
18-Oct-19 18:09:48 - SELECT * FROM etl.feedcontrol WHERE feedname ='bank'
18-Oct-19 18:09:54 - value of attendence_id is 31
18-Oct-19 18:09:54 - Job is FAILED
18-Oct-19 18:09:54 - Error occured while creating dataframe or extracting data from feedcontrol and calender tables
Traceback (most recent call last):
  File "/home/etl/ETL/Trail_code/ETL_Ingestion.py", line 58, in <module>
    feedname = feed.collect()[0]['feedname']
IndexError: list index out of range
18-Oct-19 18:09:54 - fileformat check
